e2e/__tests__/detectOpenHandles.ts

```diff
@@ -123,3 +123,25 @@ it('prints out info about open handlers from inside tests', async () => {
 
   expect(wrap(textAfterTest)).toMatchSnapshot();
 });
+
+it('prints out info about open handlers from tests with a `done` callback', async () => {
+  const {stderr} = runJest('detect-open-handles', [
+    'in-done-function',
+    '--detectOpenHandles',
+    '--forceExit',
```

Comment:
can you use `waitUntil` instead of `forceExit`?

Reply:
I think it makes more sense to match the style of the other tests which will all time out if the message is not printed.

One thing we could do in general I guess is to add a timeout to `waitUntil` and first wait for tests to complete, _then_ wait for the handle text with a timeout. But regardless, for consistency now I prefer not using `forceExit`